flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/table/descriptors/ElasticsearchValidator.java

```diff
@@ -126,4 +137,39 @@ private void validateConnectionProperties(DescriptorProperties properties) {
 		properties.validateInt(CONNECTOR_CONNECTION_MAX_RETRY_TIMEOUT, true, 1);
 		properties.validateString(CONNECTOR_CONNECTION_PATH_PREFIX, true);
 	}
+
+	/**
+	 * Parse Hosts String to list.
+	 *
+	 * <p>Hosts String format was given as following:
+	 *
+	 * <pre>
+	 *     connector.hosts = http://host_name:9092;http://host_name:9093
+	 * </pre>
+	 * @param descriptorProperties
+	 * @return
+	 */
+	public static List<ElasticsearchUpsertTableSinkBase.Host> validateAndGetHostsStr(DescriptorProperties descriptorProperties) {
+		final List<ElasticsearchUpsertTableSinkBase.Host> hostList = new ArrayList<>();
+
+		final String hostsStr = descriptorProperties.getString(CONNECTOR_HOSTS);
+		if (null == hostsStr || hostsStr.length() == 0) {
+			throw new ValidationException("Properties '" + CONNECTOR_HOSTS + "' can not be empty, but is:" + hostsStr);
```

Comment:
It is useless to print the host string when it is empty.

Reply:
ok